styles.css

```diff
@@ -3,11 +3,8 @@
 }
 
 body {
-  display: flex;
   font-family: poppins, sans-serif;
-  flex-direction: column;
-  margin: 0 auto;
-  width: 375px;
+  border: solid 1px;
```

Comment:
- [ ] There seems to be a black border around your project which is not in the Figma design, I'll suggest you remove this. :+1: :100: 

- - Screenshot:
![Selection_080](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/107632780/301369c8-1f0f-4476-b72b-dc1083862bd1)